concourse/steps/release.py

```diff
@@ -437,3 +437,14 @@ def post_to_slack(
         else:
             raise RuntimeError('Unable to get file id from Slack response')
     logger.info('successfully posted contents to slack')
+
+def invoke_post_release_callback(
```

Comment:
I don't think this wrapper adds any value, how about calling `_invoke_callback` directly in template?

Reply:
Not sure, I created this to not call underscore methods from outside. Should I remove the underscore at the same time, or are we fine calling it with an underscore?

Reply:
generally speaking you are correct, according to [PEP8](https://peps.python.org/pep-0008/#descriptive-naming-styles) a leading underscore is a "weak internal use" indicator

for concourse pipeline template please keep in mind that there is a render step.
`_invoke_callback` is rendered next to `def invoke_post_release_callback()...`

there are also other occasions where underscore functions are called in `.mako` module, whereas on source code level they are defined in corresponding `.py` module, e.g. [here](https://github.com/gardener/cc-utils/blob/22e96677b0559b83a0f1669d7f2924dbfa063e95/concourse/steps/release.mako#L424)

Reply:
done.